electrumpersonalserver/server/common.py

```diff
@@ -303,7 +303,6 @@ def get_scriptpubkeys_to_monitor(rpc, config):
                     break
                 spks_to_monitor.append(spks[0])
             wal.rewind_one(change)
-
```

Comment:
Please remove these irrelevant line changes. Or if you want to refactor then open another PR for them.